src/main/java/io/github/connorhartley/guardian/internal/detections/SpeedDetection.java

```diff
@@ -160,161 +167,257 @@ public Guardian getPlugin() {
-                    "Magic values for materials touching the player that affect the players speed which are added each tick.",
-                    materialValues, new TypeToken<Map<String, Double>>() {
-            });
+                this.configPunishmentLevels = new StorageValue<>(new StorageKey<>("punishment-levels"),
+                        "Punishments that happen when the user reaches the individual severity threshold.",
+                        punishmentLevels, new TypeToken<Map<String, Double>>() {
+                });
+
+                HashMap<String, String> punishmentProperties = new HashMap<>();
+                punishmentProperties.put("channel", "admin");
+                punishmentProperties.put("releasetime", "12096000");
+
+                this.configPunishmentProperties = new StorageValue<>(new StorageKey<>("punishment-properties"),
+                        "Properties that define certain properties for all the punishments in this detection.",
+                        punishmentProperties, new TypeToken<Map<String, String>>() {
+                });
+
+                HashMap<String, List<String>> customPunishments = new HashMap<>();
+                customPunishments.put("example", Collections.singletonList("msg %player% You have been prosecuted for illegal action!"));
+
+                this.configCustomPunishments = new StorageValue<>(new StorageKey<>("custom-punishments"),
+                        "Custom punishments that can execute custom commands.",
+                        customPunishments, new TypeToken<Map<String, List<String>>>() {
+                });
+
+                HashMap<String, Double> severityDistribution = new HashMap<>();
+                severityDistribution.put("lower", 0d);
+                severityDistribution.put("mean", 25d);
+                severityDistribution.put("standard-deviation", 15d);
+
+                this.configSeverityDistribution = new StorageValue<>(new StorageKey<>("severity-distribution"),
+                        "Normal distribution properties for calculating the over-shot value from the mean.",
+                        severityDistribution, new TypeToken<Map<String, Double>>() {
+                });
+
+                // Player Control
+
+                HashMap<String, Double> controlValues = new HashMap<>();
+                controlValues.put("sneak", 1.015);
+                controlValues.put("walk", 1.035);
+                controlValues.put("sprint", 1.065);
+                controlValues.put("fly", 1.075);
+
+                this.configControlValues = new StorageValue<>(new StorageKey<>("control-values"),
+                        "Magic values for movement the player controls that are added each tick.",
+                        controlValues, new TypeToken<Map<String, Double>>() {
+                });
+
+                // Block Speed
+
+                HashMap<String, Double> materialValues = new HashMap<>();
```

Comment:
Any need to keep all of the Maps in the class as HashMap<>? If not, might as well switch to Map<>.